warehouse/models/intermediate/gtfs/int_gtfs_rt__trip_updates_summaries.sql

```diff
@@ -53,6 +53,7 @@ int_gtfs_rt__trip_updates_summaries AS (
         MAX(trip_update_timestamp) AS max_trip_update_timestamp,
         MAX(trip_update_delay) AS max_delay,
         COUNT(DISTINCT CASE WHEN schedule_relationship = 'SKIPPED' THEN stop_id END) AS num_skipped_stops,
+        COUNT(DISTINCT CASE WHEN schedule_relationship IN ("SCHEDULED","CANCELED”,“ADDED") THEN 1 END) AS num_scheduled_canceled_added,
```

Comment:
I think you want `then stop_id` rather than `then 1` if you want to count the distinct stops; also maybe `num_scheduled_canceled_added_stops`?

Just FYI we'll have to run a full refresh of this model; by default dbt will ignore new columns ([docs](https://docs.getdbt.com/docs/build/incremental-models#what-if-the-columns-of-my-incremental-model-change)) but it is something we can change if we find ourselves doing this enough!

Also formatting nitpick: spaces between elements of the list, and single-quotes rather than double

Reply:
Thank you!